projects/cloudflare/cfssl/docker/linux/Dockerfile

```diff
@@ -0,0 +1,15 @@
+ARG BASE_IMAGE # https://gallery.ecr.aws/eks-distro-build-tooling/eks-distro-minimal-base
+FROM $BASE_IMAGE
+
+ARG TARGETARCH
+ARG TARGETOS
+
+copy _output/gitdependencies/cfssl_trust /etc/cfssl
+COPY _output/bin/cfssl/$TARGETOS-$TARGETARCH/* /usr/bin/
+COPY _output/LICENSES /LICENSES
+COPY ATTRIBUTION.txt /ATTRIBUTION.txt
+
+EXPOSE 8888
+
+ENTRYPOINT ["cfssl"]
+CMD ["--help"]
```

Comment:
mistake?

Reply:
Actually that's what they have upstream. I was kinda surprised as well.
https://github.com/cloudflare/cfssl/blob/master/Dockerfile#L14

Reply:
I guess they just assume when you are running it you are overriding the command. So odd looking tho

Reply:
Yea looks that way. So in an example use case i do see the image being used with an override
https://github.com/tinkerbell/sandbox/blob/main/deploy/compose/docker-compose.yml#L6